src/main/java/dre/task/Task.java

```diff
@@ -0,0 +1,43 @@
+package dre.task;
+
+import java.time.LocalDate;
+
+public class Task {
+    private String task;
+    private boolean done;
+    private LocalDate deadlineDate;
+
+    public Task() {
+        task = "default";
+        done = false;
+    }
+    public Task(String newTask){
+        task = newTask;
+        done = false;
+    }
+
+    public boolean isDone() {
```

Comment:
Should this method be deleted? I think it is not used anywhere, sorry if I am wrong